.env.example

```
@@ -57,6 +57,14 @@ LOG_LEVEL=INFO
 # Log format: console (human-readable) or json (structured)
 LOG_FORMAT=console
 
+# =============================================================================
+# LLM SAMPLING / DETERMINISM (Optional)
+# =============================================================================
+#
+# For paper-style reproducible benchmarks, keep these at 0.0.
+OPENAI_TEMPERATURE=0.0
+ANTHROPIC_TEMPERATURE=0.0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ANTHROPIC_TEMPERATURE key should go before the OPENAI_TEMPERATURE key

(UnorderedKey)